tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/ClientSideEncryptionTestRunner.cs

```diff
@@ -41,7 +41,9 @@ public void Run(JsonDrivenTestCase testCase)
             RequirePlatform
                 .Create()
                 .SkipWhen(SupportedOperatingSystem.Linux, SupportedTargetFramework.NetCoreApp11)
-                .SkipWhen(() => testCase.Name.Contains("gcpKMS.json"), SupportedOperatingSystem.Linux, SupportedTargetFramework.NetCoreApp21); // gcp is supported starting from netstandard2.1
+                .SkipWhen(() => testCase.Name.Contains("gcpKMS.json"), SupportedOperatingSystem.Linux, SupportedTargetFramework.NetCoreApp21) // gcp is supported starting from netstandard2.1
+                .SkipWhen(SupportedOperatingSystem.MacOS, SupportedTargetFramework.NetCoreApp11)
+                .SkipWhen(() => testCase.Name.Contains("gcpKMS.json"), SupportedOperatingSystem.MacOS, SupportedTargetFramework.NetCoreApp21); // gcp is supported starting from netstandard2.1
```

Comment:
See comment in the other PR regarding the names of the target frameworks (i.e. `NetStandard15` instead of `NetCoreApp11` etc...)